### PR TITLE
[Added] Sort by tracked achievements

### PR DESCRIPTION
**Pull request by JanGalek**
This resolving https://github.com/idiomatic/HandyNotes_Achievements/issues/18

- Added: option to turn on/off this feature (false default)
- Added: sort tracked achievements up of show list

### CODE REVIEW

**Comment by JanGalek** [comment]
@idiomatic I'm not sure if default should be `true` or `false` :)

File: HandyNotes_Achievements.lua [hunk 1]
```diff
@@ -65,14 +65,70 @@ function HNA:HandyNotesCoordsNear(c, coord)
 end
 
 
+function SortByTrackedAchievement(nodes, enabled)
```
Comment (idiomatic) [inline HandyNotes_Achievements.lua]: can `enabled` be a global variable instead?  alternatively, the caller can:

    if condition then
        nodes = SortByTrackedAchievement(nodes)
    end

and simplify the `SortByTrackedAchevements()` signature.

File: HandyNotes_Achievements.lua [hunk 2]
```diff
@@ -65,14 +65,70 @@ function HNA:HandyNotesCoordsNear(c, coord)
...
+
 function HNA:OnEnter(mapFile, nearCoord)
     tooltip = QTip:Acquire(ADDON_NAME, 2, "LEFT", "RIGHT")
     local firstRow = true
     local previousAchievementID
 
     local nodes = activeNodes[mapFile]
-    for nodeIndex = 1, #nodes, 2 do
-        local coord, row = nodes[nodeIndex], nodes[nodeIndex + 1]
+    local sortedNodes = SortByTrackedAchievement(nodes, HNA.db.profile.sort_by_tracked)
```
Comment (idiomatic) [inline HandyNotes_Achievements.lua]: `nodes` is unused after this point, and there's a slight penalty for adding another symbol.  Could you just overwrite `nodes` in this assignment, and revert the `nodes` loop?

**Comment by JanGalek** [comment]
Sorry for delay, I have much work (at work) :)

**Review by idiomatic — APPROVED** [review]

**Comment by idiomatic** [comment]
Should also add a note to the changelog for each feature.

**Comment by JanGalek** [comment]
Yes, I'll send PR for it.

**Pull request merged**